source/blog/2023-06-27-solidus-v4-1.html.erb

```diff
@@ -0,0 +1,25 @@
+---
+title: Solidus v4.1
+date: 2023-05-08
+tags: Solidus
+author: Solidus Core Team
+cover_image: /blog/2023/05/08/solidus-v4-0/cover.jpg
+---
+
+- <a class="button" href="https://github.com/solidusio/solidus/releases/tag/v4.1.0">4.1.0 on GitHub</a>
+- <a class="button button-secondary" href="https://rubygems.org/gems/solidus/versions/4.1.0">4.1.0 on RubyGems</a>
+
+Solidus v4.1 is the first minor release of the 4.x series. It contains a number of new features, bug fixes, and performance improvements.
+
+This release is relatively tiny and should be quite easy to upgrade. It starts to lay down the foundational
+work for the new Admin, which will be delivered gradually during the rest of the v4.x releases.
```

Comment:
```suggestion
work for the new Admin, which will be delivered gradually starting from one of the v4.x releases.
```

We could easily release a major before the new admin is officially delivered and fully available.